src/ConcordiumNetSdk/Types/PayloadSize.cs

```diff
@@ -0,0 +1,35 @@
+using ConcordiumNetSdk.Helpers;
+
+namespace ConcordiumNetSdk.Transactions;
+
+/// <summary>
+/// Represents a payload size.
+/// </summary>
+public struct PayloadSize
```

Comment:
I don't think we need a class for this. I would just use U32.

Reply:
Good point.